CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.8.8
```

Comment:
Can I ask you to revert `CHANGELOG.md` changes? I like to put them into `Release X version` commit.

Reply:
`## x.x.x` will be fine ?